server/game/gamesteps/triggeredabilitywindowtitles.js

```diff
@@ -1,6 +1,7 @@
 const EventToTitleFunc = {
     onCardAbilityInitiated: event => 'the effects of ' + event.source.name,
-    onCardPowerChanged: event => event.card.name + ' gaining power',
+    onCardPowerGained: event => event.card.name + ' gaining power',
+    onCardPowerMoved: event => 'power moved from ' + event.source.name + ' to ' + event.target.name,
```

Comment:
Bit nitpicky but we've been starting to use interpolation and it seems to fit here